ElvaRivle/Feature/MOwpcahY

ElvaRivle opened the pull request:
ElvaRivle/Feature/MOwpcahY

--------------------------------------------------------------------
 NOTES
 * also includes 6lWP7t30
--------------------------------------------------------------------
 MAIN
 * created protected routes
 * Sidebar navigation works
--------------------------------------------------------------------
 OTHER
 * fixed `.anticon` not showing up
 * root div now has display flex
 * Sidebar now has proper height
 --------------------------------------------------------------------
 TODO
 * everybody needs to set their div elements to `width: 100%`, because of root div display flex, #24
 * sidebar active link styling doesn't work on firefox, hopefully will work soon (devs working actively)

ElvaRivle committed:
ElvaRivle/Feature/MOwpcahY

--------------------------------------------------------------------
 NOTES
 * also includes 6lWP7t30
--------------------------------------------------------------------
 MAIN
 * created protected routes
 * Sidebar navigation works
--------------------------------------------------------------------
 OTHER
 * fixed `.anticon` not showing up
 * root div has display flex
 --------------------------------------------------------------------
 TODO
 * everybody needs to set their div elements to `width: 100%`, because of root div display flex, #24
 * sidebar active link styling doesn't work on firefox, hopefully will work soon (devs working actively)

ElvaRivle committed:
ElvaRivle/Feature/MOwpcahY

--------------------------------------------------------------------
 MAIN
 * exact true error for navlink  fixed
--------------------------------------------------------------------
 OTHER
 * /

becarselman commented on src/components/sidebar/Sidebar.scss:
```diff
@@ -92,11 +84,19 @@
+
+  text-decoration: none;
+}
+//:has doesn't work in firefox, it is actively worked on by firefox devs
```
remove uneccessary comments

becarselman commented:
Its looks okay to me if you fix bugs and also please remove unused routes for example /home, /projects because we now use /dashboard/project

ElvaRivle commented:
> Its looks okay to me if you fix bugs and also please remove unused routes for example /home, /projects because we now use /dashboard/project

Those routes are fixed in another PR, this is just sidebar navigation functionalities

ElvaRivle committed:
ElvaRivle/Feature/MOwpcahY

--------------------------------------------------------------------
MAIN
* Sidebar now has proper height
--------------------------------------------------------------------
OTHER
* /
